main/export-options.js

```diff
@@ -19,6 +19,13 @@ const exportUsageHistory = new Store({
   }
 });
 
+const fpsUsageHistory = new Store({
+  name: 'fps-usage-history',
+  defaults: {
```

Comment:
I would recommend using the `schema` option for new things as it validates the type: https://github.com/sindresorhus/electron-store#schema